app/portainer/components/index.js

```diff
@@ -1,6 +1,9 @@
 import angular from 'angular';
 
 import { CreateAccessTokenAngular } from '../views/account/CreateAccessToken';
+import { SettingsEdgeComputeAngular } from '../settings/edge-compute/SettingsEdgeCompute';
```

Comment:
I would prefer those to be declared in the settings module, vs the components module